sunpy/map/tests/test_mapbase.py

```diff
@@ -480,11 +490,50 @@ def test_rotate_invalid_order(generic_map):
 
 
 @skip_wcsaxes
+def test_as_mpl_axes_aia171(aia171_test_map):
+    import wcsaxes  # import here because of skip
```

Comment:
Would this test fail if one run the tests without `wcsaxes` installed and skipping wcsaxes?